.travis.yml

```diff
@@ -4,6 +4,11 @@ sudo: required
 services:
   - docker
 
+go:
+  - 1.6
+  - 1.7
+  - tip
```

Comment:
It would be easy to parametrize the go version in the Dockerfile, though. I'm guessing a simple `FROM golang:{{ $version }}-alpine` would do, and travis CI surely exports a GO_VERSION environment variable.

If we do that, we have to make sure that the image is only *pushed* to the registry for one of the versions (`tip` for instance)